### ISSUE TITLE
Including the ratio of unclassified reads in the ccmetagen results

### ISSUE THREAD

**Issue by permia** [vrmarcelino/CCMetagen#68]
Though the total fragment count and the mapped fragment couts can be found in the mapstat file of kma, these results seem to not appear in the ccmetagen results. To intuitively evaluate the classifying results, it's better to show  the ratio of unclassified reads in the krona results.

**Comment by vrmarcelino** [comment]
Hi

Thank you for the suggestion.
You can retrieve the proportion of unclassified reads with the extended_features flag (-ef, which also needs the mapstat output of KMA). This will show the % of classified reads in the output and helps to have an idea of the performance (but won't show it in the krona file).

**Comment by permia** [comment]
Yes. After adding the `-ef` option in kma commond and `-ef y` option in CCMetagen.py commond, there is an additional perc_map column for each template in `*_stats.csv` file.
